dev-docs/bidders/condorx.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: bidder
+title: CondorX
+description: Prebid CondorX Bidder Adapter
+media_type: native, banner
+biddercode: condorx
+pbjs: true
+tcfeu_supported: false
```

Comment:
This document is missing quite a few pieces of adapter metadata. Please fill out the metadata as documented at https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter